### DISCUSSION TITLE
Feature & Performance Compare with vim-lsp?

### DISCUSSION THREAD

**Discussion by linrongbin16** [hrsh7th/nvim-cmp#939] [General]
Hi,
Apart from that nvim-cmp is neovim only, could you compare the main difference on features and performance with vim-lsp ?

Reference: https://github.com/prabirshrestha/vim-lsp

**Comment by hrsh7th** [comment]
The nvim-cmp only achieves the `completion` that is contained in the LSP spec.
For example, nvim-cmp has the `vim-lsp` source. https://github.com/dmitmel/cmp-vim-lsp

Therefore, they are not like being compared.
